content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -383,15 +383,15 @@ Possible errors:
 
 #### <inlinecode>P3020</inlinecode>
 
-"The automatic creation of shadow databases is disabled on Azure SQL. Please set up a shadow database using the `shadowDatabaseUrl` datasource attribute.<br />Read the docs page for more details: https://pris.ly/d/migrate-shadow"
+"The automatic creation of shadow databases is disabled on Azure SQL. Please set up a shadow database using the `shadowDatabaseUrl` datasource attribute.<br />Read the docs page for more details: [https://pris.ly/d/migrate-shadow](https://pris.ly/d/migrate-shadow)"
 
 #### <inlinecode>P3021</inlinecode>
 
-"Foreign keys cannot be created on this database. Learn more how to handle this: https://pris.ly/d/migrate-no-foreign-keys"
+"Foreign keys cannot be created on this database. Learn more how to handle this: [https://pris.ly/d/migrate-no-foreign-keys](https://pris.ly/d/migrate-no-foreign-keys)"
 
 #### <inlinecode>P3022</inlinecode>
 
-"Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database. Please read more here how to handle this: https://pris.ly/d/migrate-no-direct-ddl"
+"Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database. Please read more here how to handle this: [https://pris.ly/d/migrate-no-direct-ddl]([https://pris.ly/d/migrate-no-direct-ddl)"
```

Comment:
In the deploy preview, this URL is not hyper-linked at all.... Also we need to insert "about" in "Please read more here about how to handle this:"

Reply:
To fix this link:
```suggestion
"Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database. Please read more here how to handle this: [https://pris.ly/d/migrate-no-direct-ddl](https://pris.ly/d/migrate-no-direct-ddl)"
```